tools/netsh/ebpfnetsh.rc

```diff
@@ -68,7 +68,7 @@ BEGIN
 \n                  program.\
 \n      type      - A string used as if it were a section name, to\
 \n                  determine the program type and expected attach\
-\n                  type.  If not specified, the default is xdp.\
```

Comment:
Since this is now mandatory you need to remove the `[]` around line 58

Reply:
This is not mandatory. This input is only required when:
1. section name in ELF cannot determine the program type (".text" section for example), or
2. User wants to override the program type